packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts

```diff
@@ -448,7 +448,7 @@ export interface IContextualMenuItem {
    * be used to interrupt native on click events as well. The click event should still handle
    * the commands. This should only be used in special cases when react and non-react are mixed.
    */
-  onMouseDown?: (item: IContextualMenuItem, event: any) => void;
+  onMouseDown?: (item: IContextualMenuItem, event: React.MouseEvent<HTMLElement>) => void;
```

Comment:
This could potentially cause compile errors if people have given their event param a different type locally (though it's a pretty easy fix on their end).

Reply:
That would be a bug on the consumers end that they should fix and which falls under our allowable changes re: backwards compatibility. Worth pointing out for sure tho.